deblack/deblack.py

```diff
@@ -94,6 +92,9 @@ def str2bool(v):
     parser.add_argument("input", type=str, help="input video file")
     parser.add_argument("--invert", action="store_true", help="remove nonblack instead of removing black")
     parser.add_argument("--audio", type=str2bool, default="auto", help="input video contains audio? (auto, yes, no)")
+    parser.add_argument("--black-min-duration", type=float, default=2.0, help="Minimum length of time to consider - see https://ffmpeg.org/ffmpeg-filters.html#blackdetect for more")
```

Comment:
If, instead of hard-coding the defaults, you'd prefer to have more complicated logic around passing the values to the function itself, I can do that.